### DISCUSSION TITLE
[Help] How do I use the default notebook?

### DISCUSSION THREAD

**Discussion by on3iro** [zk-org/zk#336] [Help]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

Hi,

I specified a default notebook inside my `~/.config/zk/config.toml` like so:

```toml
[notebook]

dir = "~/notes"
```

I was under the impression that this would allow me to run zk anywhere and use this notebook by default,
however I always receive the `no notebook found` error. Am I doing something wrong or am I just misinterpreting the use case for the default notebook? If it is the latter - what is it actually good for?

Thanks in advance :)

**Comment by mickael-menu** [comment]
That's correct and works for me. What exact output do you get when running `zk list` in `~/notes`?

**Reply by on3iro** [reply]
From within `~/notes` I receive a list of notes. From every other directory I get the aforementioned error

**Reply by mickael-menu** [reply]
Weird. What if you set the `ZK_NOTEBOOK_DIR = "~/notes"` env variable instead?

**Reply by on3iro** [reply]
It still fails, but not because it couldn't find the notebook, but because apparently it didn't expand `~`.
But I think you are onto something here, when checking the env variable inside another shell session it is empty.
